core/preview.ts

```diff
@@ -205,16 +205,16 @@ export async function addLink(url: string, deep = false): Promise<void> {
       if (!response.ok) {
         $links.setKey(url, { state: 'unloadable' })
       } else {
+        if (!deep) {
```

Comment:
We have a bug in `core` which creating jumping `previewCandidatesLoading` between `true` and `false`.

Not we have `false` only once in the end.